src/analyze/analyze-time-data.c

```diff
@@ -215,22 +217,44 @@ int pretty_boot_time(sd_bus *bus, char **ret) {
         return 0;
 }
 
+void related_units_free(RelatedUnits *r) {
+        if (!r)
+                return;
+
+        r->after = strv_free_erase(r->after);
```

Comment:
Why erase? These are not sensitive information.

Reply:
Honest answer: lack of documentation in the header file as to what it is for, and clangd told me an inaccurate type for `strv_free`.